pkg/cvo/cvo.go

```diff
@@ -96,32 +113,54 @@ func New(
 	eventBroadcaster.StartRecordingToSink(&coreclientsetv1.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
 
 	optr := &Operator{
-		nodename:      nodename,
-		namespace:     namespace,
-		name:          name,
-		releaseImage:  releaseImage,
+		nodename:     nodename,
+		namespace:    namespace,
+		name:         name,
+		releaseImage: releaseImage,
+		payloadDir:   overridePayloadDir,
+
+		minimumUpdateCheckInterval: 2 * time.Minute,
+
+		defaultUpstreamServer: "http://localhost:8080/graph",
```

Comment:
looks suspiciously like a test value

Reply:
Carry over from before.  We don't have our prod URLs yet.